server/src/controllers/dispositifController.ts

```diff
@@ -30,8 +28,13 @@ import {
   StructureReceiveDispositifRequest,
   UpdateDispositifPropertiesRequest,
   UpdateDispositifRequest,
-  UpdateDispositifResponse,
-} from "@refugies-info/api-types";
+  UpdateDispositifResponse
+} from "@refugies-info/api-types"
+import express from "express"
+import { NotFoundError } from "src/errors"
```

Comment:
I think this will throw an error on deployment, we need to use the relative path instead. 
Locally, it works fine with `npm run dev`, but if we do `npm run tsc && npm run start`, it does not work

Reply:
Awesome  catch @matthieu-fesselier !  🙏🏼